src/cdk/testing/test-element.ts

```diff
@@ -128,3 +132,13 @@ export interface TestElement {
   /** Checks whether the element is focused. */
   isFocused(): Promise<boolean>;
 }
+
+export function getTextWithExcludedElements(element: Element, excludeSelector: string) {
```

Comment:
Seems weird to put it in this file. I want to suggest a new file called like `dom-helpers.ts` or something, but I suspect @jelbourn might not like that 😂